types/src/on_chain_config/mod.rs

```diff
@@ -62,7 +62,6 @@ impl fmt::Display for ConfigID {
 /// State sync will panic if the value of any config in this registry is uninitialized
 pub const ON_CHAIN_CONFIG_REGISTRY: &[ConfigID] = &[
     ApprovedExecutionHashes::CONFIG_ID,
-    GasSchedule::CONFIG_ID,
```

Comment:
According to the comment above: "State sync will panic if the value of any config in this registry is uninitialized". I don't think this is gonna work with configs with multiple versions like the gas schedule -- the invariant is that one and only one version of the resource is available. Are you guys fine with me just not registering the gas schedule resources here?

Reply:
Noted. Could you track those issues for later discussion? 

Lets get forward and land now as soon as we reasonably can. There is another PR after that I thought. Actually 2.